app/views/Data/Workspace/WorkTable/index.tsx

```diff
@@ -14,10 +20,68 @@ import {
     MdOutlineLanguage,
     MdOutlineCalendarToday,
     MdOutlineLocationOn,
+    MdGrid3X3,
+    MdHdrAuto,
 } from 'react-icons/md';
 import styles from './styles.module.css';
 
-export default function WorkTable() {
+interface Props {
+    table: WorkTableQuery['table']
+}
+
+interface Column {
+    key: number;
+    label: string;
+}
+
+type Row = Record<string, string>;
+
+export default function WorkTable(props: Props) {
+    const { table } = props;
+
+    const [opened, toggleOpened] = useToggle([true, false]);
+
+    const colsKeys: string[] = table?.dataColumnStats.map((col: Column) => col.key);
```

Comment:
Is the type correct? `key` is of type `number` in `Column`.

Reply:
`key` should be string. If it is number then it's error from server side.